test/test.js

```diff
@@ -328,6 +328,25 @@ describe('Page', function() {
     }));
   });
 
+  describe('Page.setOfflineMode', function() {
+    it('should work', SX(async function() {
+      await page.setOfflineMode(true);
+      let error = null;
+      await page.goto(EMPTY_PAGE).catch(e => error = e);
+      expect(error).toBeTruthy();
```

Comment:
Can we check error message here? Also what is the error here? Timeout?

Reply:
@JoelEinbinder it's currently "Failed to navigate!", since we don't report errors for now. It won't be a timeout though: it gonna be ERR_INTERNET_DISCONNECTED

Reply:
Ok good for now I guess. Would be nice to report a more descriptive error here. Also how do I automate the dino game? :D

Reply:
> Would be nice to report a more descriptive error here

Yes, hopefully it would be easier to do with plznavigate enabled.